pyproject.toml

```diff
@@ -118,6 +118,8 @@ extra-dependencies = [
   "pandas-stubs == 2.1.4.231227",
   # For pytest types
   "pytest >= 7.4.0",
+  # For check graph generation
+  "networkx-stubs",
```

Comment:
@alejandro-salgado shouln't this requirement be in the `tests` or `default` env settings instead of the `lint` settins?

Reply:
this script is not for execution at the moment, this is is part of the lint environment only because is the only one that  interacts with the code for the moment.

In the following PR a hatch script to execute the graph visualization will be added and the dependency to the networkx library will be added to it.